crates/sui-gql-client/src/queries/filtered_full_objects.rs

```diff
@@ -0,0 +1,148 @@
+use std::collections::HashMap;
+
+use af_sui_types::{Address, Object, ObjectId};
+use futures::Stream;
+
+use super::fragments::{ObjectFilterV2, PageInfoForward};
+use crate::queries::Error;
+use crate::{extract, missing_data, scalars, schema, GraphQlClient, GraphQlResponseExt, Paged};
+
+pub(super) fn query<C: GraphQlClient>(
+    client: &C,
+    owner: Option<Address>,
+    type_: Option<String>,
+    page_size: Option<u32>,
+) -> impl Stream<Item = Result<(ObjectId, Object), Error<C::Error>>> + '_ {
```

Comment:
```suggestion
) -> impl Stream<Item = Result<Object, Error<C::Error>>> + '_ {
```
This information is redundant, caller can get it via [`Object::id`](https://docs.rs/af-sui-types/latest/af_sui_types/struct.Object.html#method.id)